src/hooks/useNpgSdk.ts

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from "react";
+import type {
+  FieldId,
+  FieldStatus,
+} from "../features/payment/components/IframeCardForm/types";
+import { getConfigOrThrow } from "../utils/config/config";
+import createBuildConfig from "../utils/buildConfig";
+
+export type SdkBuild = {
+  onChange?: (field: FieldId, fieldStatus: FieldStatus) => void;
+  onReadyForPayment?: () => void;
+  onPaymentComplete?: () => void;
+  onPaymentRedirect?: (urlRedirect: string) => void;
+  onBuildError: () => void;
+};
+
+const noop = () => {
+  // noop
+};
+
+export const useNpgSdk = ({
+  onChange = () => null,
+  onReadyForPayment = () => null,
+  onPaymentComplete = () => null,
+  onPaymentRedirect = () => null,
+  onBuildError,
+}: SdkBuild) => {
+  const [sdkReady, setSdkReady] = useState(false);
+
+  const createBuild = () => {
+    try {
+      // THIS is mandatory cause the Build class is defined in the external library called NPG SDK
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      const newBuild = new Build(
+        createBuildConfig({
```

Comment:
newBuild is declared but never used, I think we can just instantiate. (just new Build)

Reply:
Removed the unused declaration